include/mysql/service_wsrep.h

```diff
@@ -172,6 +173,7 @@ extern struct wsrep_service_st {
 #define wsrep_drupal_282555_workaround get_wsrep_drupal_282555_workaround()
 #define wsrep_recovery get_wsrep_recovery()
 #define wsrep_protocol_version get_wsrep_protocol_version()
+#define wsrep_thd_set_wsrep_aborter(T) wsrep_service->wsrep_thd_set_wsrep_aborter_func(T1, T2)
```

Comment:
Why does this macro only take (and discard) 1 parameter? What are the `T1` and `T2`?

Reply:
This seems to be used only when wsrep is compiled as dynamic plugin I will fix it.